db.go

```diff
@@ -16,13 +16,13 @@ func OpenDBWithDriver(driver string, dbstring string) (*sql.DB, error) {
 	case "mssql":
 		driver = "sqlserver"
 	case "redshift":
-		driver = "postgres"
+		driver = "pgx"
```

Comment:
The alternative is to remove this override entirely, which I think is more correct.

In the [CLI we use `pgx`](https://github.com/pressly/goose/blob/c6706ad68cdc7e06b921447ac75514eb75a657c7/cmd/goose/main.go#L121-L123) since we're solely responsible for picking the driver, but in this case a user might be using lib/pq and an override to pgx would fail.

Probably best to just remove it and leave this up to the caller.

Reply:
Removed in https://github.com/pressly/goose/pull/567/commits/e3dbb9b1fdb53e4c380ce4209f92f940a06b6c30

I believe this should fix the issue for existing users who want to use lib/pq

Also added an override back to the CLI code to use pgx driver.